requirements.txt

```diff
@@ -1,5 +1,5 @@
 boost
```

Comment:
@adithya-22 If we are specifying version numbers for cryptography and pyopenssl can we do it for passlib and pycparser as well?

Reply:
okay, sure.